commit def78819ba3a6c7dddc8677b73f4220aa07ac45f (refs/changes/63/2363/1)
Author: Elijah Zupancic <elijah@zupancic.name>
Date:   2017-08-09T15:52:38-07:00 (2 years, 2 months ago)
    
    MANTA-3370 Added Go SDK and Hadoop FS to SDK List

diff --git a/docs/user-guide/sdks.md b/docs/user-guide/sdks.md
index 5b906ac..a86714e 100644
--- a/docs/user-guide/sdks.md
+++ b/docs/user-guide/sdks.md
@@ -101,6 +101,17 @@ it add the following to your project's pom.xml:
 
 You can find the source for Java SDK at [https://github.com/joyent/java-manta](https://github.com/joyent/java-manta).
 
+# Hadoop FileSystem Driver
+
+Hadoop can natively access Manta over the network. Most major Hadoop FileSystem
+features are supported. This is a community driven project, so updates to it 
+are dependent upon community involvement. The driver is available as a 
+stand-alone jar file that can be dropped into a Hadoop or Apache Drill 
+installation.
+
+You can download the jar directory from the [releases page](https://github.com/joyent/hadoop-manta/releases) 
+on the [project's github page](https://github.com/joyent/hadoop-manta) 
+or directly from Maven Central.
 
 # PHP SDK
 
@@ -114,3 +125,11 @@ and can be installed using [Composer](https://getcomposer.org/):
 It has been tested in PHP 5.6, PHP 7.0 and HHVM.
 
 You can find the source for PHP SDK at [https://github.com/joyent/php-manta](https://github.com/joyent/php-manta).
+
+# Golang SDK
+
+The Go SDK for Manta is currently in active development. It is a combination SDK
+that provides support for Triton CloudAPI operations in addition to Manta API
+operations.
+
+You can find the source for the Go Manta SDK at [https://github.com/joyent/triton-go](https://github.com/joyent/triton-go).
